CMakeLists.txt

```diff
@@ -293,6 +293,9 @@ if (TILEDB_TESTS)
   add_dependencies(tests unit_range_subset)
   add_dependencies(tests unit_range)
   add_dependencies(tests unit_mgc_dict)
+  if (TRUE) #(TILEDB_BUILDING_WEBP)
```

Comment:
- This needs to be an option
-  It should not be defaulted to TRUE

Reply:
Understood.  Trying to run it through CI before optionalizing it.  (Those locations/conditions are intended to be changed.)
Where, in what builds, if any, do you want it to be active?

Reply:
It should be enabled in one build for each platform, at least. Preferably one of the faster ones.